website/routes.py

```diff
@@ -337,6 +349,36 @@ def make_url_map(app):
                     notemplate
                 )
             ])
+            EXTERNAL_EMBER_APPS['ember_osf_web']['asset_prefixes'] = [
+                'assets',
```

Comment:
Should these be configurable? If they're not meant to run on Prod, that would let us set `EMBER_ASSET_PREFIXES=None` on the prod config just to ensure no route pollution.

Reply:
This is meant for Prod too. When the entire frontend is an Ember app, we will just serve its dist/ directory and not need to route anything through flask. These prefixes need to be reserved and not used as routes. I would like to go ahead and mimc the normal Ember setup now on all environments. This change will allow us, for example, to start serving image assets from the Ember app.